Jenkinsfile

```diff
@@ -174,6 +174,25 @@ pipeline {
             echo 'Skipping Kafka Streams archetype test for Java 21'
           }
         }
+
+        stage('PowerPC') {
```

Comment:
The KIP states that this build will run nightly. Can you clarify which part of this stage does the scheduling?

Reply:
As per my knowledge the job must be configured in Jenkins Job to run nightly, correct me if wrong?

Reply:
I have updated the code to run nightly.